<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xFlow</class>
 <widget class="QWidget" name="xFlow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <widget class="xFlow::NodeEditorRuler" name="widgetHRuler" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>15</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="xFlow::NodeEditorRuler" name="widgetVRuler" native="true">
     <property name="minimumSize">
      <size>
       <width>15</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>15</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="xFlow::NodeEditor" name="widgetNodeEditor" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>xFlow::NodeEditor</class>
   <extends>QWidget</extends>
   <header location="global">x/flow/nodeeditor/nodeeditor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>xFlow::NodeEditorRuler</class>
   <extends>QWidget</extends>
   <header location="global">x/flow/nodeeditor/nodeeditorruler.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
